spec/models/news_item_spec.rb

```diff
@@ -1,5 +1,61 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe NewsItem, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "validation" do
+    it { should validate_presence_of(:title) }
+    it { should validate_length_of(:title).is_at_most(255) }
+    it { should validate_presence_of(:body) }
+    it { should validate_length_of(:body).is_at_most(4000) }
+    context "for state" do
```

Comment:
Yes, I wrote it with the shoulda-matcher originally, but it kept failing because it would try to test that the enum state field can accept a symbol input for its setter, but would return a string output. It didn't like the asymmetry and complained about it as bad API. I wish I'd discover a way to relax its strictness, but then I realized it's not worth spending too much time forcing it to work for a few validations, and just wrote my version of inclusion testing.